mesonbuild/build.py

```diff
@@ -1304,6 +1304,9 @@ def __init__(self, name, subdir, subproject, is_cross, sources, objects, environ
         # Only linkwithable if using export_dynamic
         self.is_linkwithable = self.export_dynamic
 
+    def desc(self):
```

Comment:
I think this name is a bit on the cryptic side. We should try to come up with a more descriptive name here.